ui/src/pages/updates/update-log/UpdateLog.hooks.ts

```diff
@@ -17,7 +17,7 @@ export const useUpdateLog = (authToken: string) => {
 
       return getUpdateStatus({ authToken, logOffset });
     },
-    refetchInterval: 500,
+    refetchInterval: ({ state }) => (state.data?.done ? 0 : 500),
```

Comment:
IMO `500ms` is too frequent, I'd recommend we set it to 1s, which should be good enough to not degrade the UX.

Reply:
@matejkubinec FYI ^

Reply:
Makes sense, updated.